### DISCUSSION TITLE
Anyway to run external processes outside sandbox?

### DISCUSSION THREAD

**Discussion by Dr-TSNG** [sandboxie-plus/Sandboxie#1878] [Documents and notes]
I install a program inside the sandbox and start it. What I wish is when it opens external programs (such as explorer or a PDF reader to open a file in the shared directory), the external process should not run in the sandbox. Is this possible?

**Comment by isaak654** [comment]
* #1500  - See also [BreakoutProcess](https://sandboxie-plus.github.io/sandboxie-docs/Content/BreakoutProcess.html) and [BreakoutFolder](https://sandboxie-plus.github.io/sandboxie-docs/Content/BreakoutFolder.html) documentation.
